### PR TITLE
Include fluent-plugin-detect-exceptions in plugin_gems.

### PR DESCRIPTION
**Pull request by thomasschickinger**

### CODE REVIEW

File: plugin_gems.rb [hunk 1]
```diff
@@ -19,3 +19,4 @@
 download "fluent-plugin-rewrite-tag-filter", "1.5.5"
 download "fluent-plugin-td-monitoring", "0.2.2"
 download "fluent-plugin-google-cloud", "0.5.2"
+download "fluent-plugin-detect-exceptions", "0.0.1"
```
Comment (igorpeshansky) [inline plugin_gems.rb]: Didn't you just bump the version to 0.0.2?

Comment (thomasschickinger) [inline plugin_gems.rb]: Not yet. But you are right, ideally the new Gem should go in first and then I need to bump this version number here before submitting this change.

Comment (thomasschickinger) [inline plugin_gems.rb]: Bumped to the newest version 0.0.3 that is about to be submitted to Rubygems.org.

File: plugin_gems.rb [hunk 2]
```diff
@@ -19,3 +19,5 @@
 download "fluent-plugin-rewrite-tag-filter", "1.5.5"
 download "fluent-plugin-td-monitoring", "0.2.2"
 download "fluent-plugin-google-cloud", "0.5.2"
+download "fluent-plugin-detect-exceptions", "0.0.3"
+>>>>>>> f367b32e17df1353ca49df6ed5a2c0a5b54a8945
```
Comment (igorpeshansky) [inline plugin_gems.rb]: Can you please remove the conflict marker?

Comment (thomasschickinger) [inline plugin_gems.rb]: Apologies for overlooking that.

**Comment by igorpeshansky** [comment]
LGTM :shipit:

**Comment by rbuskens** [comment]
LGTM.

**Pull request merged**